velox/connectors/hive/CMakeLists.txt

```diff
@@ -21,6 +21,11 @@ add_library(velox_hive_partition_function HivePartitionFunction.cpp)
 
 target_link_libraries(velox_hive_partition_function velox_core)
 
+add_library(velox_hive_write_protocol OBJECT HiveWriteProtocol.cpp)
```

Comment:
Any particular reason not to include HiveWriteProtocol.cpp into velox_hive_connector ? Why "OBJECT"?

Reply:
Revising. I'd like to include it into velox_hive_connector, which is easier. Just saw HivePartitionFunction.cpp was in a separate item